### ISSUE TITLE
Suggestion - Possibility to create custom characters tables

### ISSUE THREAD

**Issue by dvgniele** [damienleroy/PowerAccent#54]
I was thinking about, what if every user can edit the characters table?
I mean, just editing a json file, everyone can create a table for each letter. 

This could be usefull also because in this way there is no necessary compile whenever you want to add a new language.
It could also be useful for the community, so that everyone could share their own custom pattern.

For example:
 - if a user can select the desired language, there should be a "custom language" where he can create the preferred characters table.

Just like it is in the theme settings for the new Windows Terminal.

**Comment by jojo553** [comment]
yes please. being able to add and remove accented characters for each character would be amazing. i don't need all accented characters, just one or 2 for particular characters, which are not available. i hope this is available asap. it would be very helpful.

**Comment by damienleroy** [comment]
It is not the first time people asking me this feature. It is something I could do. But it will take some time. Thanks for your patience!
